### ISSUE TITLE
Suggestion: Merging this with the base engine

### ISSUE THREAD

**Issue by moxie-coder** [CamtheKirby/VsDave-Modable#9]
I might consider merging this with Dave engine, since I'm the (official) maintainer done by Erizur himself because this has plenty of stuff Dave Engine itself lacks currently

**Comment by CamLikesKirby** [comment]
Wait there's an official engine now, is there a repository?

**Comment by CamLikesKirby** [comment]
That I can see

**Comment by moxie-coder** [comment]
> Wait there's an official engine now, is there a repository?

https://github.com/Erizur/FNF-DaveEngine there's been one for a long while now, but it's pretty unfinished, and this may help with that

**Comment by CamLikesKirby** [comment]
It looks cool but there's a lot of bugs and unfinished stuff with this engine rn so it may not be the best time to merge it, but I am going to work on this engine soon when I have the time

**Comment by moxie-coder** [comment]
> It looks cool but there's a lot of bugs and unfinished stuff with this engine rn so it may not be the best time to merge it, but I am going to work on this engine soon when I have the time

yeah that's what I said, I've been working on it for a while now and I'm trying to fix them

**Comment by CamLikesKirby** [comment]
I was talking about my engine my bad

**Comment by moxie-coder** [comment]
> I was talking about my engine my bad

oop, np
